Formula/atomist-cli.rb

```diff
@@ -0,0 +1,23 @@
+require "language/node"
+
+class AtomistCli < Formula
+  desc "The Atomist CLI"
+  homepage "https://github.com/atomist/cli#readme"
+  url "https://registry.npmjs.org/@atomist/cli/-/@atomist/cli-1.0.3.tgz"
+  sha256 "41753f416340a17f5674cf698bf6381d999767a187857e784c8488c8b03777c3"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+    bash_completion.install "#{libexec}/lib/node_modules/@atomist/cli/assets/bash_completion/atomist"
+  end
+
+  test do
+    assert_predicate bin/"atomist", :exist?
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
Functionality tests added below.